staging/src/k8s.io/apiserver/pkg/admission/chain.go

```diff
@@ -16,22 +16,38 @@ limitations under the License.
 
 package admission
 
-// chainAdmissionHandler is an instance of admission.Interface that performs admission control using a chain of admission handlers
-type chainAdmissionHandler []Interface
+import "time"
+
+// chainAdmissionHandler is an instance of admission.Interface that performs admission control using
+// a chain of admission handlers
+type chainAdmissionHandler []NamedHandler
 
 // NewChainHandler creates a new chain handler from an array of handlers. Used for testing.
-func NewChainHandler(handlers ...Interface) chainAdmissionHandler {
+func NewChainHandler(handlers ...NamedHandler) chainAdmissionHandler {
```

Comment:
I think you'll need to update a couple unit tests: 
https://github.com/kubernetes/kubernetes/blob/01f205adf51ca34a527f2bf978e5606d1bb3fd59/plugin/pkg/admission/namespace/autoprovision/admission_test.go#L139
https://github.com/kubernetes/kubernetes/blob/01f205adf51ca34a527f2bf978e5606d1bb3fd59/plugin/pkg/admission/persistentvolume/label/admission_test.go#L81

(See https://github.com/kubernetes/kubernetes/pull/55086/files#diff-97778d709e3ef8a841d3749d2c16072a)

Reply:
Thanks @tallclair! I've fixed the tests, making heavy use of the work you did for #55086.